draft-ietf-scitt-architecture.md

```diff
@@ -165,29 +170,34 @@ Issuers can register their Signed Statements on one or more Transparency Service
-The shipping company may add information about the transport environment of the Artifact.
-Compliance Auditors may provide information about their compliance assessment of the Artifact.
-Security companies may publish vulnerability information about an Artifact.
-The original manufacturer may subsequently provide additional information about the manufacturing process they discovered after the Artifact left the factory.
-Some of these parties may publish information about their analysis or use of an Artifact.
-
-SCITT provides a way for Relying Parties to obtain this information in a way that is "transparent", that is, parties cannot lie about the information that they publish without it being detected.
-SCITT achieves this by having producers publish information in a Transparency Service, where Relying Parties can check the information.
+This document defines an architecture, a base set of extensible message structures and associated flows to make signed data items transparent via verifiable data structures maintained by corresponding transparency services.
+The goal of the transparency enabled by the Supply Chain Integrity, Transparency, and Trust (SCITT) architecture is to enhance auditability and accountability for signed data items (statements) that are about supply chain commodities (artifacts).
+Registering signed statements with a transparency service is akin to a notarization procedure.
+Transparency services therefore act like notaries that each record (register) a single, linear and irrevocable history of statements that were made.
+How these statements are managed or stored---beside the specified registration procedure---is out-of-scope of this document.
+Earlier, similar approaches for specific classes of artifacts exist, such as Certificate Transparency {{-CT}}.
+The SCITT approach follows a more generic paradigm than previous approaches.
+In this SCITT core architecture, the signed statements registered are opaque to the transparency services that act as a notary.
+This "content-agnostic" approach allows SCITT transparency services to be either integrated in existing solutions or to be an initial part of new emerging systems.
+Extensibility is a vital feature of the SCITT architecture, so that requirements from various applications can be accommodated while always ensuring interoperability with respect to registration procedures and corresponding auditability and accountability.
+For simplicity, the scope of this document is limited to use cases originating from the software supply chain domain, but the specification defined is applicable to any other type of supply chain statements (also referred to as value-add graphs), for example, statements about hardware supply chains.
+
+This document also defines message structures for signed statements and defines a profile for COSE receipts {{-RECEIPTS}}, i.e., countersignatures with metadata).
```

Comment:
I thought we were avoiding "countersignatures" 🤷‍♂️

Reply:
I intended to quote Russ, ngl. We do not have to express that here, but this the story why receipts are called receipts in a very concise nutshell.

will check for "needsfixing" in about 2h